src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -82,4 +82,16 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    default String getPrintableString(Printable... printables){
+        StringBuilder builder = new StringBuilder();
+        for (Printable p : printables){
+            builder.append(p.getPrintableString());
+            builder.append(" ");
+        }
+        return builder.toString().trim();
+    }
```

Comment:
No need to `trim()` again as all joined strings have been trimmed previously already.